src/elixirSenseClient.ts

```diff
@@ -50,6 +52,13 @@ export class ElixirSenseClient {
     handleConnect() {
         this.resetBuffer(0, 4);
         console.log(`[vscode-elixir] ElixirSense client connected on ${this.host}:${this.port}`);
+        this.send('version', {})
```

Comment:
awaiting a change in https://github.com/msaraiva/elixir_sense/pull/8 , to use this endpoint

Reply:
So, its probably best to wait for that and not pull this PR for now?

Reply:
It is in! I'm going to whip up a quick pr to update the submodule to elixirsense v1.0.0